Userland/Services/SpiceAgent/Message.h

```diff
@@ -218,6 +231,30 @@ class FileTransferStartMessage : public Message {
     Metadata m_metadata;
 };
 
+// Sent to indicate the status of the current file transfer
+class FileTransferStatusMessage : public Message {
+public:
+    static FileTransferStatusMessage create(u32 id, FileTransferStatus status);
+    static ErrorOr<FileTransferStatusMessage> read_from_stream(AK::Stream& stream);
+
+    ErrorOr<void> write_to_stream(AK::Stream& stream) override;
+    ErrorOr<String> debug_description() override;
+
+    u32 id() const { return m_id; };
+    FileTransferStatus const& status() { return m_status; };
+
+private:
+    FileTransferStatusMessage(u32 id, FileTransferStatus status)
+        : Message(Type::FileTransferStatus)
+        , m_id(id)
+        , m_status(status)
+    {
+    }
+
+    u32 m_id;
```

Comment:
Please initialize primitive data members :)
`u32 m_id { 0 };` should suffice here.